idaes/core/base/property_base.py

```diff
@@ -761,11 +761,22 @@ def calculate_scaling_factors(self):
         super().calculate_scaling_factors()
         # Get scaling factor defaults, if no scaling factor set
         for v in self.component_data_objects(
-            (Constraint, Var, Expression), descend_into=False
+            (Var, Expression), descend_into=False
         ):
             if iscale.get_scaling_factor(v) is None:  # don't replace if set
                 name = v.getname().split("[")[0]
                 index = v.index()
                 sf = self.config.parameters.get_default_scaling(name, index)
                 if sf is not None:
                     iscale.set_scaling_factor(v, sf)
+
+        for con in self.component_data_objects(Constraint, descend_into=False):
+            # if con.name == 'stripper_section.stripper.liquid_phase.properties[0.0,0.8].appr_to_true_species[Liq,CO2]':
+            #     import pdb; pdb.set_trace()
+            if iscale.get_constraint_transform_applied_scaling_factor(con) is None:
+                name = con.getname().split("[")[0]
+                index = con.index()
+                sf = self.config.parameters.get_default_scaling(name, index)
+                if sf is not None:
+                    iscale.constraint_scaling_transform(con, sf, overwrite=False)
```

Comment:
This is a fairly significant change to what this method does - we probably need to discuss the implications of this.

Reply:
Thinking a bit more (and repeating a comment from Slack for posterity) - I think this is the wrong way to go.

IPOPT should be considered a special case, not the general case, in which case we need to set the scaling suffix for the constraints and then let the user decide what to do with them. IPOPT users can use the `constraint_scaling_transformation` function if they wish, but the generic workflow would use the Pyomo scaling transformation (which relies on having the suffixes).

Reply:
Removing the default status of IPOPT and `constraint_scaling_transform` in the long run is fine, but it's also a complete revolution in how scaling is treated in IDAES models, which make heavy use of `constraint_scaling_transform`. If this change is reverted, I'll just as likely use `constraint_scaling_transform` somewhere else to do the desired scaling: probably in `electrolyte_states.py`.

Reply:
It is fine to still use `constraint_scaling_transform`, but it should not be done in the base classes (i.e. anything in `idaes.core.base`). Ideally, we would only do this in the model level `calculate_scaling_factors` method.

The point being this is not something we want baked in deep as it will be harder to change later - ideally we want to limit the use of this to the `calculate_scaling_factors` methods so that when we make the transition, we only need to change those (or if we are lucky, just switch to new methods and delete those as they are no longer needed).